[2025-07-30T22:03:19.736+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_medallon.crea_esquema_publico manual__2025-07-30T22:03:15+00:00 [queued]>
[2025-07-30T22:03:19.821+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_medallon.crea_esquema_publico manual__2025-07-30T22:03:15+00:00 [queued]>
[2025-07-30T22:03:19.822+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-07-30T22:03:19.881+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): crea_esquema_publico> on 2025-07-30 22:03:15+00:00
[2025-07-30T22:03:19.886+0000] {standard_task_runner.py:60} INFO - Started process 71 to run task
[2025-07-30T22:03:19.892+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'pipeline_medallon', 'crea_esquema_publico', 'manual__2025-07-30T22:03:15+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_medallon_dag.py', '--cfg-path', '/tmp/tmp8yze1wfm']
[2025-07-30T22:03:19.895+0000] {standard_task_runner.py:88} INFO - Job 59: Subtask crea_esquema_publico
[2025-07-30T22:03:20.011+0000] {task_command.py:423} INFO - Running <TaskInstance: pipeline_medallon.crea_esquema_publico manual__2025-07-30T22:03:15+00:00 [running]> on host 70eaa744a05c
[2025-07-30T22:03:20.159+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mateo.biasotto' AIRFLOW_CTX_DAG_ID='pipeline_medallon' AIRFLOW_CTX_TASK_ID='crea_esquema_publico' AIRFLOW_CTX_EXECUTION_DATE='2025-07-30T22:03:15+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-30T22:03:15+00:00'
[2025-07-30T22:03:20.161+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-07-30T22:03:20.162+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/airflow/scripts/01_crea_esquema_publico.py']
[2025-07-30T22:03:20.169+0000] {subprocess.py:86} INFO - Output:
[2025-07-30T22:03:20.412+0000] {subprocess.py:93} INFO - üîó Conectado a PostgreSQL en postgres:5432/medallon_db
[2025-07-30T22:03:20.414+0000] {subprocess.py:93} INFO - ‚ñ∂Ô∏è Ejecutando: /opt/airflow/sql/01_crear_tablas_public.sql
[2025-07-30T22:03:20.415+0000] {subprocess.py:93} INFO - ‚ö†Ô∏è Error en la sentencia:
[2025-07-30T22:03:20.416+0000] {subprocess.py:93} INFO - CREATE TABLE IF NOT EXISTS public.bulk_fact_ventas (
[2025-07-30T22:03:20.416+0000] {subprocess.py:93} INFO -     taligent_id  VARCHAR PRIMARY KEY, -- clave concatenada order_id-product_id
[2025-07-30T22:03:20.417+0000] {subprocess.py:93} INFO -     order_id     VARCHAR,
[2025-07-30T22:03:20.418+0000] {subprocess.py:93} INFO -     order_date   DATE,
[2025-07-30T22:03:20.418+0000] {subprocess.py:93} INFO -     ship_date    DATE,
[2025-07-30T22:03:20.419+0000] {subprocess.py:93} INFO -     customer_id  VARCHAR REFERENCES public.dim_cliente(customer_id),
[2025-07-30T22:03:20.419+0000] {subprocess.py:93} INFO -     product_id   VARCHAR REFERENCES public.dim_producto(product_id),
[2025-07-30T22:03:20.420+0000] {subprocess.py:93} INFO -     sales        NUMERIC,
[2025-07-30T22:03:20.421+0000] {subprocess.py:93} INFO -     quantity     INT,
[2025-07-30T22:03:20.421+0000] {subprocess.py:93} INFO -     discount     NUMERIC,
[2025-07-30T22:03:20.422+0000] {subprocess.py:93} INFO -     profit       NUMERIC
[2025-07-30T22:03:20.422+0000] {subprocess.py:93} INFO - )
[2025-07-30T22:03:20.423+0000] {subprocess.py:93} INFO - relation "public.dim_cliente" does not exist
[2025-07-30T22:03:20.423+0000] {subprocess.py:93} INFO - 
[2025-07-30T22:03:20.424+0000] {subprocess.py:93} INFO - ‚ö†Ô∏è Error en la sentencia:
[2025-07-30T22:03:20.424+0000] {subprocess.py:93} INFO - -- ===============================================
[2025-07-30T22:03:20.425+0000] {subprocess.py:93} INFO - -- TABLAS FINALES (PUBLIC)
[2025-07-30T22:03:20.425+0000] {subprocess.py:93} INFO - -- ===============================================
[2025-07-30T22:03:20.425+0000] {subprocess.py:93} INFO - 
[2025-07-30T22:03:20.426+0000] {subprocess.py:93} INFO - -- Clientes (dimensi√≥n)
[2025-07-30T22:03:20.426+0000] {subprocess.py:93} INFO - CREATE TABLE IF NOT EXISTS public.dim_cliente (
[2025-07-30T22:03:20.427+0000] {subprocess.py:93} INFO -     customer_id   VARCHAR PRIMARY KEY,
[2025-07-30T22:03:20.427+0000] {subprocess.py:93} INFO -     customer_name VARCHAR,
[2025-07-30T22:03:20.428+0000] {subprocess.py:93} INFO -     segment       VARCHAR,
[2025-07-30T22:03:20.429+0000] {subprocess.py:93} INFO -     country       VARCHAR,
[2025-07-30T22:03:20.430+0000] {subprocess.py:93} INFO -     city          VARCHAR,
[2025-07-30T22:03:20.430+0000] {subprocess.py:93} INFO -     state         VARCHAR,
[2025-07-30T22:03:20.431+0000] {subprocess.py:93} INFO -     postal_code   VARCHAR,
[2025-07-30T22:03:20.431+0000] {subprocess.py:93} INFO -     region        VARCHAR
[2025-07-30T22:03:20.432+0000] {subprocess.py:93} INFO - )
[2025-07-30T22:03:20.432+0000] {subprocess.py:93} INFO - current transaction is aborted, commands ignored until end of transaction block
[2025-07-30T22:03:20.432+0000] {subprocess.py:93} INFO - 
[2025-07-30T22:03:20.433+0000] {subprocess.py:93} INFO - ‚ö†Ô∏è Error en la sentencia:
[2025-07-30T22:03:20.433+0000] {subprocess.py:93} INFO - -- Productos (dimensi√≥n)
[2025-07-30T22:03:20.434+0000] {subprocess.py:93} INFO - CREATE TABLE IF NOT EXISTS public.dim_producto (
[2025-07-30T22:03:20.434+0000] {subprocess.py:93} INFO -     product_id    VARCHAR PRIMARY KEY,
[2025-07-30T22:03:20.435+0000] {subprocess.py:93} INFO -     product_name  VARCHAR,
[2025-07-30T22:03:20.436+0000] {subprocess.py:93} INFO -     category      VARCHAR,
[2025-07-30T22:03:20.437+0000] {subprocess.py:93} INFO -     sub_category  VARCHAR
[2025-07-30T22:03:20.439+0000] {subprocess.py:93} INFO - )
[2025-07-30T22:03:20.440+0000] {subprocess.py:93} INFO - current transaction is aborted, commands ignored until end of transaction block
[2025-07-30T22:03:20.440+0000] {subprocess.py:93} INFO - 
[2025-07-30T22:03:20.441+0000] {subprocess.py:93} INFO - ‚ö†Ô∏è Error en la sentencia:
[2025-07-30T22:03:20.442+0000] {subprocess.py:93} INFO - -- Ventas (hechos)
[2025-07-30T22:03:20.443+0000] {subprocess.py:93} INFO - CREATE TABLE IF NOT EXISTS public.fact_ventas (
[2025-07-30T22:03:20.443+0000] {subprocess.py:93} INFO -     taligent_id  VARCHAR PRIMARY KEY, -- clave concatenada order_id-product_id
[2025-07-30T22:03:20.444+0000] {subprocess.py:93} INFO -     order_id     VARCHAR,
[2025-07-30T22:03:20.445+0000] {subprocess.py:93} INFO -     order_date   DATE,
[2025-07-30T22:03:20.445+0000] {subprocess.py:93} INFO -     ship_date    DATE,
[2025-07-30T22:03:20.446+0000] {subprocess.py:93} INFO -     customer_id  VARCHAR REFERENCES public.dim_cliente(customer_id),
[2025-07-30T22:03:20.447+0000] {subprocess.py:93} INFO -     product_id   VARCHAR REFERENCES public.dim_producto(product_id),
[2025-07-30T22:03:20.447+0000] {subprocess.py:93} INFO -     sales        NUMERIC,
[2025-07-30T22:03:20.448+0000] {subprocess.py:93} INFO -     quantity     INT,
[2025-07-30T22:03:20.449+0000] {subprocess.py:93} INFO -     discount     NUMERIC,
[2025-07-30T22:03:20.449+0000] {subprocess.py:93} INFO -     profit       NUMERIC
[2025-07-30T22:03:20.450+0000] {subprocess.py:93} INFO - )
[2025-07-30T22:03:20.450+0000] {subprocess.py:93} INFO - current transaction is aborted, commands ignored until end of transaction block
[2025-07-30T22:03:20.451+0000] {subprocess.py:93} INFO - 
[2025-07-30T22:03:20.451+0000] {subprocess.py:93} INFO - ‚úÖ Script ejecutado correctamente
[2025-07-30T22:03:20.452+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-07-30T22:03:20.526+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=pipeline_medallon, task_id=crea_esquema_publico, execution_date=20250730T220315, start_date=20250730T220319, end_date=20250730T220320
[2025-07-30T22:03:20.628+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-07-30T22:03:20.686+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
